apps/browser/qml/pages/components/HistoryList.qml

```diff
@@ -32,29 +34,8 @@ SilicaListView {
         showDeleteButton: view.showDeleteButton
         onMenuOpenChanged: view.menuClosed = !menuOpen
 
-        Component {
+        HistoryContextMenu {
```

Comment:
Could this be added directly inside `HistoryItem.qml`? It doesn't look like `HistoryItem` will ever need to be used without the context menu.

Reply:
Done